src/Component/Header/indx.js

```diff
@@ -15,7 +15,7 @@ import Toolbar from '@mui/material/Toolbar';
 import Typography from '@mui/material/Typography';
 import Button from '@mui/material/Button';
 import style from './style.module.css';
-
+import { Link } from 'react-router-dom';
```

Comment:
Link should be import from mui, please fix 
`import Link from '@mui/material/Link';`